build.wake

```diff
@@ -1,5 +1,5 @@
-def installDir = simplify "build/{here}"
-global def freedomDevicetreeToolsInstallDir = "build/{here}"
+def installDir ="build/freedom-devicetree-tools"
```

Comment:
This is a silly nitpick, but could you add the missing space between = and "?

Reply:
Happy to, it looks bad